From: Custom TLS Fingerprint <noreply@example.com>
Date: Mon, 1 Jan 2024 00:00:00 +0000
Subject: [PATCH 1/4] net: add TLSFingerprintConfig source files to net/BUILD.gn

In Chromium 100+ there is no standalone net/ssl/BUILD.gn.  All ssl/
sources are compiled as part of the monolithic component("net") target
in net/BUILD.gn.

Adds net/ssl/tls_fingerprint_config.{h,cc} to the existing ssl/ sources
list in alphabetical order (after threaded_ssl_private_key, before
storage_access_api).
---
 net/BUILD.gn | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/net/BUILD.gn b/net/BUILD.gn
index 00000000..11111111 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -1112,6 +1112,8 @@
     "ssl/ssl_platform_key_util.h",
     "ssl/threaded_ssl_private_key.cc",
     "ssl/threaded_ssl_private_key.h",
+    "ssl/tls_fingerprint_config.cc",
+    "ssl/tls_fingerprint_config.h",
     "storage_access_api/status.h",
     "url_request/clear_site_data.cc",
     "url_request/clear_site_data.h",
